pkg/policy/parser.go

```diff
@@ -53,8 +53,9 @@ var policySchema = &hcl.BodySchema{
 	},
 }
 
-func DecodePolicy(body hcl.Body, diags hcl.Diagnostics, basePath string) (*Policy, hcl.Diagnostics) {
+func decodePolicy(body hcl.Body, diags hcl.Diagnostics, basePath string) (*Policy, hcl.Diagnostics) {
 	content, contentDiags := body.Content(policyWrapperSchema)
+	diags = append(diags, contentDiags...)
 	if contentDiags.HasErrors() {
 		return nil, diags
 	}
```

Comment:
```suggestion
	content, diags := body.Content(policyWrapperSchema)
	if diags.HasErrors() {
		return nil, diags
	}
```

Reply:
This way we would lose `diags` that were already passed to us as an argument. How knows what's inside and what errors it might have.

Reply:
So you're suggesting to change fn signature also, to remove diags?

Reply:
@disq I'm not sure I understand :)

Reply:
comment was to @roneli we just happened to post at the same minute :)
I agree with your points.

Reply:
Removed unnecessary diags argument